http-client/mhs/asynchronous-express-message-pattern/Return-information-from-ebxml-fault-returned-from-spine-in-original-post-request-when-sync-async-requested.http

```diff
@@ -0,0 +1,16 @@
+
+
+@PATIENT_NHS_NUMBER = 9689177923
+@DISSENTOVERRIDE = dissentOverride
+# @name Return_information_from_ebxml_fault_returned_from_spine_in_original_post_request_when_sync_async_requested
+POST http://localhost HTTP/1.1
+interaction-id: QUPC_IN160101UK05
+message-id: 7AA57E38-8B20-4AE0-9E73-B9B0C0C42BDA
```

Comment:
Anyway of generating new UUID on request?

Reply:
Only possible using an extension to generate one manually, will add extension and instructions to readme